src/odemis/acq/align/spot.py

```diff
@@ -32,7 +32,8 @@
 from odemis.acq.align.autofocus import AcquireNoBackground, MTD_EXHAUSTIVE
 from odemis.dataio import tiff
 from odemis.util import executeAsyncTask
-from odemis.util.spot import FindCenterCoordinates, GridPoints, MaximaFind, EstimateLatticeConstant
+from odemis.util.spot import FindCenterCoordinates, MaximaFind, EstimateLatticeConstant
+from odemis.util.spot import GridPointsLegacy as GridPoints
```

Comment:
No, this is asking for the reader to become mad. Either update the (single) line that use this function to use the new name, or adjust it to use the new function.